src/mtcnn/stage2.ts

```diff
@@ -56,10 +56,10 @@ export async function stage2(
 
     const regions = indicesNms.map(idx =>
       new MtcnnBox(
-        rnetOuts[indices[idx]].regions.get(0, 0),
-        rnetOuts[indices[idx]].regions.get(0, 1),
-        rnetOuts[indices[idx]].regions.get(0, 2),
-        rnetOuts[indices[idx]].regions.get(0, 3)
+        rnetOuts[indices[idx]].regions.arraySync()[0][0],
+        rnetOuts[indices[idx]].regions.arraySync()[0][1],
+        rnetOuts[indices[idx]].regions.arraySync()[0][2],
+        rnetOuts[indices[idx]].regions.arraySync()[0][3]
```

Comment:
Better call arraySync once outside of the map callback as you did [here](https://github.com/justadudewhohacks/face-api.js/pull/246/files#diff-cb18491d4c23fc57b660e5a7b2287623R50).

Reply:
Yep. But is imposible outside de map because var idx .